frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -530,8 +532,22 @@ function CreateTreeButton() {
 }
 
 function ChangeBrushSizeButton() {
+  const dispatch = useDispatch();
   const brushSize = useSelector((state: OxalisState) => state.userConfiguration.brushSize);
-  const maximumBrushSize = useSelector((state: OxalisState) => getMaximumBrushSize(state));
+  const [isBrushSizePopoverOpen, setIsBrushSizePopoverOpen] = useState(false);
+  const maximumButtonBrushSize = useSelector((state: OxalisState) => getMaximumBrushSize(state));
+  const mediumButtonBrushSize = calculateMediumBrushSize(maximumButtonBrushSize);
+  const minimumButtonBrushSize = Math.max(userSettings.brushSize.minimum, 10); // TODO unsure whether that makes sense across the board
```

Comment:
A brush smaller than 10vx is hardly visible for the dataset I used to test (test-agglomerate-file without zooming in), so I thought this makes sense but wasn't sure. See screenshots attached for visibility of brush sizes 5vx and 10vx with said settings
![Screenshot from 2023-05-30 16-28-44](https://github.com/scalableminds/webknossos/assets/57364771/314f89b6-5a87-424a-9e10-c5b31b4a7b5c)
![Screenshot from 2023-05-30 16-28-52](https://github.com/scalableminds/webknossos/assets/57364771/ec5a1c54-9a78-44f1-9e47-2b3cc0909fd1)

Reply:
> A brush smaller than 10vx is hardly visible for the dataset I used to test (test-agglomerate-file without zooming in), so I thought this makes sense but wasn't sure.

I'd vote to not add any clamping, because it can be confusing to the user. Especially, when zooming very far in, this would be very unintuitive in my opinion. If the brush size is so small, that one cannot see it, so be it :)

Reply:
but would this make sense for the default sizes or should the small preset simply be the smallest size possible?

Reply:
I'm not sure if I understood your question correctly, but I'd change the small brush size preset from 1 to 10 vx, since that is more common in my opinion.